test/integration/test_create_dss_docker_image.py

```diff
@@ -64,16 +75,18 @@ def test_jupyterlab(dss_docker_container):
     """"
     Test that jupyterlab is configured properly
     """
-    jupyter_command = (
-        "/root/jupyterenv/bin/jupyter-lab"
-        " --notebook-dir=/root/notebooks"
-        " --no-browser"
-        " --allow-root"
-    )
+    # jupyter_command = (
+    #     "/root/jupyterenv/bin/jupyter-lab"
+    #     " --notebook-dir=/root/notebooks"
+    #     " --no-browser"
+    #     " --allow-root"
+    # )
     container = dss_docker_container
-    container.exec_run(jupyter_command, detach=True)
-    container.reload()
+    # container.exec_run(jupyter_command, detach=True)
+    # container.reload()
```

Comment:
can be removed

Reply:
Sure I just left it for review. See next push.